### PR TITLE
WIP: Add database

### PR DESCRIPTION
**Pull request by BLUTCH1991**

### CODE REVIEW

File: config/database.php [hunk 1]
```diff
@@ -41,11 +41,11 @@
 
         'mysql' => [
             'driver' => 'mysql',
-            'host' => env('DB_HOST', '127.0.0.1'),
+            'host' => env('DB_HOST', '192.168.10.10'),
```
Comment (florianbouchet) [inline config/database.php]: Please change all DB informations with the following for CleverCloud : https://github.com/CleverCloud/doc.clever-cloud.com/blob/4c1d790efc94031b70b740909d395fe200b03928/runtimes/php/tutorial-laravel.md#configure-your-database

Comment (waxzce) [reply config/database.php]: :-)

File: app/Category.php [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Category extends Model
```
Comment (florianbouchet) [inline app/Category.php]: Please change `Category` to `LessonCategory` (and rename the file)

File: app/Content.php [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Content extends Model
```
Comment (florianbouchet) [inline app/Content.php]: Please change `Content` to `LessonContent` (and rename the file)

File: app/Role.php [hunk 4]
```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Role extends Model
```
Comment (florianbouchet) [inline app/Role.php]: Please change `Role` to `UserRole` (and rename the file)

File: app/Support.php [hunk 5]
```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Support extends Model
```
Comment (florianbouchet) [inline app/Support.php]: Please change `Support` to `LessonSupport` (and rename the file)

**Pull request merged**